deploy/ocp/manifests/0.13.0/0000_50_olm_07-olm-operator.deployment.yaml

```diff
@@ -67,8 +67,11 @@ spec:
             requests:
               cpu: 10m
               memory: 160Mi
-            
-          
+            limits:
+              cpu: 400m
+              memory: 400Mi
+
+
```

Comment:
Was this generated by an automated process? If not, please remove empty newlines.